src/na/zoned/decomposer/Decomposer.cpp

```
@@ -0,0 +1,278 @@
+ * Copyright (c) 2025 - 2026 Munich Quantum Software Company GmbH
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * Licensed under the MIT License
+ */
+
+//
+// Created by cpsch on 11.12.2025.
+//
+#include "na/zoned/decomposer/decomposer.hpp"
```

Cppcheck (2.19.0)

[information] Include file

(missingInclude)